projects/RabbitMQ.Client/client/api/IConnection.cs

```diff
@@ -336,9 +336,9 @@ public interface IConnection : INetworkConnection, IDisposable
         void Close(ushort reasonCode, string reasonText, TimeSpan timeout);
```

Comment:
Given that some many things are IO bound wouldnt we also want to adjust the other methods like these to use value task?

Reply:
I'd think so.